hardware_interface/src/fake_components/generic_system.cpp

```diff
@@ -37,9 +37,9 @@ return_type GenericSystem::configure(const hardware_interface::HardwareInfo & in
   // check if to create fake command interface for sensor
   auto it = info_.hardware_parameters.find("fake_sensor_commands");
   if (it != info_.hardware_parameters.end()) {
-    fake_sensor_command_interfaces = it->second == "true";
+    fake_sensor_command_interfaces_ = it->second == "true" || it->second == "True";
```

Comment:
I propose adding `parse_bool()` function to encapsulate this logic and it could be reused by other components.
https://github.com/ros-controls/ros2_control/blob/master/hardware_interface/include/hardware_interface/component_parser.hpp is the place to add this

Reply:
I would continue thus this comment. I will just add a TODO in the code.